app/src/main/java/com/example/parkingsystem_kotlin/mvp/view/ActivityView.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.parkingsystem_kotlin.mvp.view
```

Comment:
put this into a base package inside the view package

Reply:
done, thanks.